windows/src/desktop/help/common/os.md

```diff
@@ -2,23 +2,13 @@
 title: What operating systems does Keyman support?
 ---
 
-Keyman fully supports 32-bit *and* 64-bit versions of the following
+Keyman fully supports 64-bit versions of the following
```

Comment:
32-bit win10 still works

note: win11 amd doesn't work

Reply:
> note: win11 amd doesn't work

Keyman doesn't work on amd or doesn't work on ARM64?


> We need the corresponding Windows Server editions listed here

On this page too, right?

Reply:
duh. s/amd/arm64/.... because keyman runs in-process we need native arm64 build to work with any arm64 apps, unlike most x64 apps which run fine in emulation.

Reply:
> > We need the corresponding Windows Server editions listed here
> 
> On this page too, right?

yup